journal/modernized/24-chapter-21.adoc

```diff
@@ -779,3 +787,6 @@ I have not therefore demanded the money.
 Health to you and your son in your (two) voyaging.
 
 Na Daniella Wira.
+
+--
+//not sure if there are two signatures here? 
```

Comment:
yeah that's super weird. i checked in the original, and i'm not sure, it's almost like a post-script.  so, let's just leave it as is for now.